### ISSUE TITLE
YouTube-dl cannot convert media

### ISSUE THREAD

**Issue by ElDuderino80** [ish-app/ish#675]
When using YT-DL to extract audio from a video the operation fails without errors when trying to produce an M4a file. When trying to convert to mp3, there’s an error in VBRQuantize (See screen shot for full error)

The Webm version successfully downloads each time, it’s simply the conversion that seems to fail. The failed files do show up in /root as does the successful webm.

**Comment by saagarjha** [comment]
I think you might be missing a screenshot?

**Comment by ElDuderino80** [comment]
> I think you might be missing a screenshot?

Yeah, I’m on the iPad so I couldn’t figure out how to select it to add to report, had to upload to imgur and edit original.

**Comment by cowpod** [comment]
make sure that you have the necessary libraries installed

**Comment by dullbananas** [comment]
I tried using youtube-dl recently and it also failed for me
